code/game/turfs/simulated/walls_indestructible.dm

```diff
@@ -79,6 +79,25 @@
 	icon = 'icons/turf/walls/hierophant_wall.dmi'
 	icon_state = "wall"
 	smoothing_flags = SMOOTH_CORNERS
+	baseturf = /turf/simulated/floor/indestructible/hierophant/two
+
+/turf/simulated/wall/indestructible/hierophant/Initialize(mapload)
+	. = ..()
+	GLOB.hierophant_walls += src
+
+/turf/simulated/wall/indestructible/hierophant/BeforeChange()
+	GLOB.hierophant_walls -= src
+	return ..()
+
+GLOBAL_LIST_INIT(hierophant_walls, list())
+
+/turf/simulated/wall/indestructible/hierophant/proc/collapse()
+	if(prob(15))
+		visible_message("<span class='warning'>[src] starts to rumble and groan as the lights fade on it, and it begins to collapse to rubble!</span>",\
+		"<span class='warning'>You hear metal groaning and tearing!</span>")
```

Comment:
if someone is observing the hierophant arena when stuff is starting to collapse, does this spam their chat at all?

Reply:
Depends on rng. They get bunched into one message, so you might see the message x3, but since it has no variation, it won't spam chat.